src/main/java/org/jabref/logic/importer/fetcher/AdvancedSearchConfig.java

```diff
@@ -0,0 +1,128 @@
+package org.jabref.logic.importer.fetcher;
+
+import org.jabref.model.strings.StringUtil;
+
+public class AdvancedSearchConfig {
+    private final String defaultField;
+    private final String author;
+    private final String title;
+    private final int fromYear;
+    private final int toYear;
+    private final String journal;
+
+    AdvancedSearchConfig(String defaultField, String author, String title, int fromYear, int toYear, String journal) {
+        this.defaultField = defaultField;
+        this.author = author;
+        this.title = title;
+        this.fromYear = fromYear;
+        this.toYear = toYear;
+        this.journal = journal;
+    }
+
+    public boolean areTextSearchFieldsEmpty() {
```

Comment:
```suggestion
    public boolean textSearchFieldsAreEmpty() {
```

I think, this is the most used form, isn't it?